### PR TITLE
fix: should send egg-ready when app/agent restarted after launched

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
测试用例写得很不错。

**Comment by fengmk2** [comment]
1.12.1

File: test/master.test.js [hunk 1]
```diff
@@ -322,9 +322,8 @@ describe('test/master.test.js', () => {
     let app;
     let readyMsg;
 
-    after(() => app.close());
-
     before(() => {
+      mm.env('default');
```
Comment (atian25) [inline test/master.test.js]: 为啥要 default ?

Comment (popomore) [reply test/master.test.js]: unittest 不会 refork

**Pull request merged**